app/views/aspects/_aspect_dropdown.mobile.haml

```diff
@@ -0,0 +1,36 @@
+-# Note: all_aspects is a global in the ApplicationController
+:ruby
+  dropdown_css = {"data-toggle" => "dropdown"}
+  if current_user.getting_started?
+    dropdown_css[:title] = popover_with_close_html("2. #{t('shared.public_explain.control_your_audience')}")
+    dropdown_css["data-content"] = t("shared.public_explain.visibility_dropdown")
+  end
```

Comment:
I went through the sign up flow on mobile and even reaching the publisher page already had `getting_started` be `false` on my new user. Manually setting it back to `true` showed for this to have no effect. Let's drop this.